cmake/Findglad.cmake

```diff
@@ -53,7 +53,8 @@ if(glad_FOUND)
     COMMENT "Generating GLAD"
   )
 
-  add_library(glad "${glad_SOURCES}")
+  add_library(glad STATIC "${glad_SOURCES}")
+  set_target_properties(glad PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
```

Comment:
We need to enable PIC now?

Reply:
yes, because this change also affects Ubuntu.